webapp/view/profile.html

```diff
@@ -0,0 +1,58 @@
+<div class="row">
+    <div class="col-sm-3">
+        Name:
+    </div>
+    <div class="col-sm-9">
+        <?php echo $viewAccount; ?>
+    </div>
+</div>
+
+<div class="row">
+    <div class="col-sm-3">
+        Role:
+    </div>
+    <div class="col-sm-9">
+        <?php
+        switch($viewAccount) {
+            case $viewAccount instanceof Mediator:
+                echo 'Mediator';
+                break;
+            case $viewAccount instanceof MediationCentre:
+                echo 'Mediation Centre';
+                break;
+            case $viewAccount instanceof Agent:
+                echo 'Agent';
+                break;
+            case $viewAccount instanceof LawFirm:
+                echo 'Law Firm';
+                break;
+            default:
+                echo 'unknown';
```

Comment:
And then check for any other instance where we're querying the account class just to echo the account type.